op-program/client/l2/engineapi/block_processor.go

```diff
@@ -56,12 +56,23 @@ func NewBlockProcessorFromPayloadAttributes(provider BlockDataProvider, parent c
 		ParentBeaconRoot: attrs.ParentBeaconBlockRoot,
 	}
 	if attrs.EIP1559Params != nil {
-		d, e := eip1559.DecodeHolocene1559Params(attrs.EIP1559Params[:])
-		if d == 0 {
-			d = provider.Config().BaseFeeChangeDenominator(header.Time)
-			e = provider.Config().ElasticityMultiplier()
+		if provider.Config().IsJovian(header.Time) {
+			d, e, m := eip1559.DecodeJovian1559Params(attrs.EIP1559Params[:])
+			if d == 0 {
+				d = provider.Config().BaseFeeChangeDenominator(header.Time)
+				e = provider.Config().ElasticityMultiplier()
+				m = 20 // Default MinBaseFeeLog2 to 20 when not configured
```

Comment:
What should the default value be for minBaseFeeLog2 pre-Jovian?

Reply:
I initially thought zero would do the trick, but if we're running the same `CalcBaseFee` on all the historical blocks, I think the base fee could actually go down to zero (versus `2^0 = 1`). Probably needs more thought.